lib/session.js

```diff
@@ -35,11 +35,12 @@ class Session {
   toJSON() {
     const obj = {};
 
-    Object.keys(this).forEach(key => {
-      if (key === 'isNew') return;
-      if (key[0] === '_') return;
-      obj[key] = this[key];
-    });
+    Object.keys(this)
+      .filter(key => key !== 'isNew')
```

Comment:
we need tow more traversal with filter and doesn't improve any readability or maintainability.